cpu/stm32f0/periph/gpio.c

```diff
@@ -195,7 +202,7 @@ void isr_exti(void)
     for (size_t i = 0; i < EXTI_NUMOF; i++) {
```

Comment:
Sounds valid, seems like I have miss-interpreted the ref manual -> will fix this